monochart/templates/ingress.yaml

```diff
@@ -35,6 +35,9 @@ metadata:
     ingressName: {{ $name }}
   name: {{ include "common.fullname" $root }}-{{ $name }}
 spec:
+{{- if semverCompare ">=1.19" $root.Capabilities.KubeVersion.Version }}
+  ingressClassName: {{ $ingress.className }}
```

Comment:
This would mean that if our Kubernetes is new enough, this becomes a required parameter does it not?

I would imagine we either want to additional condition on `$ingress.ClasName` being defined in the incoming values, or set a `| default "ingress-nginx"` in the resultant value, so that this would not be a breaking change.